acceptance/lib/helper.rb

```diff
@@ -290,8 +290,7 @@ def https_request(url, request_method, cert, key, body = nil)
   end
 
   def hup_server(host = master, timeout = 30)
-    pidfile = on(host, 'puppet master --configprint rundir').stdout.chomp + '/puppetserver' 
-    pid = on(host, "cat #{pidfile}").stdout.chomp
+    pid = on(host, "pgrep -f #{options['puppetservice']}").stdout.chomp
```

Comment:
I think we might need to just do 'puppetserver' instead of `options['puppetservice']` here.  I think the name of the jar file in the Java process contains "puppetserver" and not "pe-puppetserver", both for OSS and PE.